packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -58,9 +59,26 @@ const AccountSwitcher = props => {
     React.useEffect(() => {
         const vrtc_loginid = props.account_list.find(account => account.is_virtual).loginid;
 
-        getExchangeRate(props.accounts[vrtc_loginid].currency, props.obj_total_balance.currency).then(res =>
-            setExchangedRate(res)
-        );
+        if (props.accounts[vrtc_loginid].currency !== props.obj_total_balance.currency) {
+            getExchangeRate(props.accounts[vrtc_loginid].currency, props.obj_total_balance.currency).then(res => {
+                setExchangedRate(res);
+            });
+
+            const mt5_demo_currency = props.mt5_login_list.find(account => isDemo(account))?.currency;
+            const dxtrade_demo_currency = props.dxtrade_accounts_list.find(account => isDemo(account))?.currency;
```

Comment:
This can be moved outside of `useEffect` hook so that it can be used by the other `useEffect` hook